SAPHana/ra/SAPHanaController

```diff
@@ -3043,6 +3105,7 @@ function saphana_monitor_clone() {
             fi
         fi
     fi
+    logger -t "fhlpdebug:" "saphana_monitor_clone() end with rc=$rc"
```

Comment:
all these 'logger -t "fhlpdebug:" ' lines above are intended to stay in the source? Or they are only temporary available for debug purposes?

Reply:
I just like to have them for a one-time test package and then we should decide which one we keep and which one we remove again, is that ok for first? We needed these log-entries to identify that WaitforStarted really came back to fast and that we really hit partial nodes in that phase. So I would like to keep them and remove them after a first test. They should not come to the official package.

Reply:
ok, understand. Thanks